### PR TITLE
Add CITATION.cff

### PR DESCRIPTION
**Pull request by alejoe91**

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+  - family-names: Baker
+    given-names: Cody
```
Comment (CodyCBakerPhD) [inline CITATION.cff:5]:
```suggestion
    given-names: Cody 
    orcid: https://orcid.org/0000-0002-0829-4790
```

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - family-names: Tauffer
+    given-names: Luiz 
+  - family-names: Buccino
+    given-names: Alessio Paolo 
+    orcid: https://orcid.org/0000-0003-3661-527X
+  - family-names: Dichter 
+    given-names: Benjamin
+    orcid: https://orcid.org/0000-0001-5725-6910
+title: "Nwb-conversion-tools"
+version: 0.9.4
```
Comment (CodyCBakerPhD) [inline CITATION.cff:19]: @alejoe91 Any way we can explicitly link this to the actual current version? Might get annoying to always have to bump both at the same time, and always possible to forget to do so

**Comment by CodyCBakerPhD** [comment]
@h-mayorquin @luiztauffer @Saksham20 Check your names for spelling and other preferences; if you have an orcid ID you can set it here too

File: CITATION.cff (hunk 2 above)
Comment (alejoe91) [reply CITATION.cff:20]: I haven't found anything automatically unfortunately :(

Comment (CodyCBakerPhD) [reply CITATION.cff:20]:
```suggestion
version: 0.9.9
```

Bummer; requesting manual bump then

Comment (CodyCBakerPhD) [reply CITATION.cff:20]: I mean, I guess we could have a workflow script or something that checks it as a part of our auto-publish and auto-generates or modifies this file prior to packaging. Not a huge priority though

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+date-released: 2021-11-18
```
Comment (CodyCBakerPhD) [inline CITATION.cff:21]:
```suggestion
date-released: 2022-2-3
```
Assuming we are doing this for the upcoming release

File: CITATION.cff [hunk 4]
```diff
@@ -0,0 +1,22 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+  - family-names: Baker
+    given-names: Cody 
+    orcid: https://orcid.org/0000-0002-0829-4790
+  - family-names: Sharda
+    given-names: Saksham 
+  - family-names: Mayorquin
+    given-names: Heberto
```
Comment (h-mayorquin) [inline CITATION.cff:10]:
```suggestion
    given-names: Heberto 
    orcid: https://orcid.org/0000-0002-5937-7537
```

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**